ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -410,13 +438,11 @@ abstract contract CoreRelayerDelivery is CoreRelayerBase, IWormholeRelayerDelive
 
     //Refund the relayer (their extra funds) + (the amount that the relayer spent on gas)
     //  + (the users refund if that refund didn't succeed)
-    Wei relayerRefundAmount = (
-      msgValue() - deliveryInstruction.receiverValueTarget - deliveryInstruction.maximumRefundTarget
-    ) + (deliveryInstruction.maximumRefundTarget - transactionFeeRefundAmount)
+    Wei relayerRefundAmount = 
```

Comment:
This is a cursed ternary. Is this here because of stackTooDeep? If not, please break this into multiple steps with named intermediate vars.

Also cleanup the comments surrounding this